lib/push-client.js

```diff
@@ -1,38 +1,6 @@
 /** @format */
 
 import { listen } from 'push-receiver';
-import request from 'request-promise';
-
-/**
- * Marks a push token as approved on WPCOM API
- * @param {String} pushToken push token that recieved via a push notification on an app
- * @param {String} bearerToken user bearer token to be used with the API
- * @returns {Promise} a request result promise
- */
-export const approvePushToken = ( pushToken, bearerToken ) =>
```

Comment:
This method is used here: https://github.com/Automattic/wp-e2e-tests/blob/master/specs/wp-log-in-out-spec.js#L167